composer.json

```diff
@@ -239,15 +239,15 @@
         "drupal/crop": "^2.0",
         "drupal/focal_point": "~1.0-beta6",
         "drupal/media_entity_video": "^2.0",
-        "drupal/video": "1.2",
+        "drupal/video": "^1.4",
         "drupal/easy_breadcrumb": "^1.8",
         "drupal/ckeditor_bootstrap_buttons": "^1.1",
         "drupal/fontyourface": "^3.2",
         "drupal/css_editor": "^1.2",
         "drupal/domain": "1.0-alpha14",
         "drupal/domain_theme_switch": "^1.4",
         "drupal/social_feed_fetcher": "^1.0",
-        "symfony/dom-crawler": "~2.8|~3.0|~4.3",
+        "symfony/dom-crawler": "~4.2",
```

Comment:
Keep old version, please
Needed for support of step by step upgrade path

Reply:
Done